### ISSUE TITLE
Is there a way to “normalize” the light condition for a group of images?

### ISSUE THREAD

**Issue by yang-ruixin** [Wanggcong/StyleLight#6]
Dear authors,

Is there a way to “normalize” the light condition for a group of images?

For example, in face recognition, I have 1000 faces. For some faces, there is strong light from upper left, for some other faces, there is strong light from both upper left and right. For some faces, light is very strong, and for others light is very weak.

I want to get rid of the light effect, can I have an average or normalized light condition for all faces?

Thanks in advance.

**Comment by jxl0131** [comment]
> 

Maybe you can use “Intrinsic decomposition” on human face to remove the Light influence.

**Comment by jxl0131** [comment]
> Dear authors,
> 
> Is there a way to “normalize” the light condition for a group of images?
> 
> For example, in face recognition, I have 1000 faces. For some faces, there is strong light from upper left, for some other faces, there is strong light from both upper left and right. For some faces, light is very strong, and for others light is very weak.
> 
> I want to get rid of the light effect, can I have an average or normalized light condition for all faces?
> 
> Thanks in advance.

Maybe you can use “Intrinsic decomposition” on human face to remove the Light influence.

**Comment by Wanggcong** [comment]
> 

@yang-ruixin Thank you for your question. Your question may be another problem like delighting or albedo estimation. I recommend some works to you.

- [1] Controllable Light Diffusion for Portraits, Siggraph 2023, [paper link](https://arxiv.org/pdf/2305.04745.pdf).
- [2] Total Relighting: Learning to Relight Portraits for Background Replacement, [project page](https://augmentedperception.github.io/total_relighting/)

Although the results are amazing, their group never released code, which make it hard to follow. Anyway, I think some other public code of delighting or albedo estimation of the references of [1][2] would motivate your research.

@jxl0131 Thank you for your suggestion.

**Comment by yang-ruixin** [comment]
@Wanggcong @jxl0131 Thank you both for your kind reply.
